.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: go
 go:
-  - 1.7.x
+  - 1.7
+  - 1.8
   - master
 addons:
   apt:
     sources:
       - sourceline: 'ppa:masterminds/glide'
     packages:
       - glide
-script: make test
+before_install:
+  - go get github.com/mattn/goveralls
+script:
+  - make vendor && $HOME/gopath/bin/goveralls -service=travis-ci -ignore "internal/internal.go,internal/public.pb.go,internal/private.pb.go"
```

Comment:
As far as I see at: https://coveralls.io/builds/11529694 it doesn't include anything from vendor.